### PR TITLE
Feature/add conjured items

### PR DESCRIPTION
**Pull request by teenie-quaggard**
- add conjured items to production code
- create two pending tests for edge cases not yet dealt with (where an item is created with either a negative or 50+ quality number)

### CODE REVIEW

File: spec/gilded_rose_spec.rb [hunk 1]
```diff
@@ -29,12 +29,17 @@
...
     it "quality should never be negative" do
-      items = [Item.new("foo", 0, 0), Item.new("foo", 0, -1)]
+      items = [Item.new("foo", 0, 0)]
+      GildedRose.new(items).update_quality()
+      GildedRose.new(items).update_quality()
+      expect(items[0].quality).to eq (0)
+    end
+
+    xit "if item is provided a negative quality value, it will return a quality of 0" do
+      items = [Item.new("foo", 0, -1)]
```
Comment (ellehallal) [inline spec/gilded_rose_spec.rb:40]: If `foo` is replaced with an item name (for example, `aged brie`, `backstage passes`), will the behaviour be the same?

File: spec/gilded_rose_spec.rb [hunk 2]
```diff
@@ -74,13 +83,12 @@
...
     end
   
-    # NEED: to deal with conjured items
-    # it "conjured items degrade in Quality twice as fast as normal items" do
-    #   items = [Item.new("conjured", 50, 50), Item.new("conjured", 0, 50)]
-    #   GildedRose.new(items).update_quality()
-    #   expect(items[0].quality).to eq (48)
-    #   expect(items[1].quality).to eq (46)
-    # end
+    it "conjured items degrade in Quality twice as fast as normal items" do
```
Comment (ellehallal) [inline spec/gilded_rose_spec.rb:86]: This is a great test 👍🏽 What are your thoughts on creating separate variables for each conjured item for readability? For example:
```
conjured_item = Item.new("conjured", 50, 50)
conjured_item_expired = Item.new("conjured", -1, 50)
items = [conjured_item, conjured_item_expired]
``` 
or something along those lines?

**Review by ellehallal — COMMENTED** [review]
Looks great, Kristina! I've left a couple of small comments. Reach out if you have any questions

File: spec/gilded_rose_spec.rb (hunk 2 above)
Comment (teenie-quaggard) [reply spec/gilded_rose_spec.rb:86]: That's a great idea! I will add that to my incoming PR. Thanks @ellehallal!

File: spec/gilded_rose_spec.rb (hunk 1 above)
Comment (teenie-quaggard) [reply spec/gilded_rose_spec.rb:40]: I see where you are going with this - thanks you!
